projects/igniteui-angular/src/lib/data-operations/paging-state.interface.ts

```diff
@@ -7,6 +7,7 @@ export enum PagingError {
 export declare interface IPagingState {
     index: number;
     recordsPerPage: number;
+    /* blazorSuppress */
```

Comment:
Going to assume that `metadata` is not even meant for public consumption? Can't even tell what those are supposed to do from looking at them anyway.